test/indexer/test_hca_indexer.py

```diff
@@ -60,11 +60,15 @@ def setUpModule():
 
 class TestHCAIndexer(IndexerTestCase):
 
-    def _get_all_hits(self):
+    def _get_all_hits(self, verify_lists_sorted=True):
```

Comment:
If `verify_lists_sorted` is never passed as `False` we should eliminate it. I was expecting it to get in the way of some of the tests but I guess I was wrong.